### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by athai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,20 @@ To set up a dev instance of StreamWebs, copy ``Dockerfile.env.dist`` to
 ``Dockerfile.env``. Then, run the following:
 ```
 $ sudo docker-compose build
-$ sudo docker-compose run web
+$ sudo docker-compose up
```
Comment (ElijahCaine) [inline README.md]: I would replace all instances of `sudo docker-compose` with just `docker-compose`. People should probably know if they need to run docker commands as root or not.

**Comment by ElijahCaine** [comment]
Pending the feedback about `sudo` I have, I'd give this a +1

**Comment by rensmere** [comment]
Hi Aileen,

It is sometimes hard to know if any of these emails are directed at me.  Maybe I can be removed from the overall workflow and then I can just be emailed directly if there are things that apply to me specifically?  Thanks.

Renee

--
Renee O'Neill
Programs Coordinator
The Science and Math Investigative Learning Experiences (SMILE) Program and
Oregon Sea Grant StreamWebs Program
Oregon State University
(541) 737-5675 or (541) 737-2747
http://smile.oregonstate.edu<http://smile.oregonstate.edu/>
http://streamwebs.org<http://streamwebs.org/>

**Comment by athai** [comment]
@rensmere I think you'll have to unwatch the repository yourself if you don't want to see the notifications. The button to do so is in the top right hand corner.

**Comment by Kennric** [comment]
+1

**Comment by matthewrsj** [comment]
late to the party, but +1

**Comment by ElijahCaine** [comment]
@leian7 https://github.com/settings/notifications and https://github.com/settings/emails might help your email problem :smiley:

**Comment by leian7** [comment]
+1

File: README.md [hunk 2]
```diff
@@ -7,12 +7,15 @@ application.
...
 ```
-$ sudo docker-compose build
-$ sudo docker-compose up
+$ docker-compose build
+$ docker-compose up
 ```
 
-``sudo docker-compose build`` will build the necessary containers while
-``sudo docker-compose up`` runs the application.
+``docker-compose build`` will build the necessary containers while
```
Comment (ElijahCaine) [inline README.md]: while what?

**Comment by ElijahCaine** [comment]
+1 looks good to me. The only edit I have is very small.

**Pull request merged**